src/routes/pages.js

```diff
@@ -1,34 +1,13 @@
 const express = require('express');
+const pageloader = require('../util/loadPages');
 
 const router = express.Router();
 
 /* GET specific page. */
 router.get('/:page_id', (req, res) => {
-  if (req.params.page_id === 'correct_page_id') {
-    const page = {
-      id: '3fa85f64-5717-4562-b3fc-2c963f66afa6',
-      type: 'info',
-      sections: [
-        {
-          type: 'text',
-          content: 'viva la dijkstra',
-        },
-        {
-          type: 'image',
-          content: 'https://i.redd.it/tfugj4n3l6ez.png',
-        },
-        {
-          type: 'animation',
-          content: 'https://www.youtube.com/watch?v=k6Es9Qrh4Ao',
-        },
-      ],
-      problem: {
-        type: 'code',
-        data: {},
-      },
-    };
-    res.json(page);
-    // res.send(page);
+  if (pageloader.hasPage(req.params.page_id) === true || req.params.page_id === 'correct_page_id') {
```

Comment:
It would be better to use actual pages ids in our system at the moment, and the 'correct_page_id' statement can possibly be removed.
Accordingly, the test suites of load pages can also be changed to an actual page id.

Reply:
I have changed to the actual page id in test suites and remove the 'correct_page_id' statement.